rosdep/python.yaml

```
@@ -7104,6 +7104,9 @@ python3-nose-yanc:
   debian: [python3-nose-yanc]
   openembedded: [python3-nose-yanc@meta-ros-common]
   ubuntu: [python3-nose-yanc]
+python3-numba:
+  debian: [python3-numba]
+  ubuntu: [python3-numba]
```

This package is not currently available in Ubuntu Jammy. Since it is available in Debian Unstable I'm not sure whether using a wildcard and nulling out Jammy or explicitly listing the Ubuntu LTS releases that do contain the package is more likely to remain accurate going forward. I've opted for the more conservative approach.

```suggestion
  ubuntu: 
    bionic: [python3-numba]
    focal: [python3-numba]
```